### ISSUE TITLE
Improve docs around adding channels.

### ISSUE THREAD

**Issue by brad-jones** [prefix-dev/pixi#593]
### Problem description

Take this for example.
![image](https://github.com/prefix-dev/pixi/assets/2754772/6b69e99e-502c-4bfd-9b0e-9c8ed133d0a5)

It should actually be something more like:

```
pixi project channel add https://repo.prefix.dev/brads-forge
pixi add lefthook
```

But ideally a single command like the micromamba version.

On this page https://prefix.dev/channels/brads-forge there should also be some hint as to how to add the channel.

Here https://pixi.sh/cli/#project-channel-add although I recognize now that there is an example of how to add a prefix.dev channel that wasn't immediately obvious to me. Perhaps something like `pixi project channel add https://repo.prefix.dev/<YOUR-CHANNEL-NAME-HERE>` may have helped.

The upload example shows: `pixi upload repo.prefix.dev/my_channel my_package.conda`. _Turns out that is incorrect too. see: https://github.com/prefix-dev/pixi/issues/248_

So I figured `pixi project channel add repo.prefix.dev/brads-forge` should work but it didn't.

**Comment by ruben-arts** [comment]
Yeah we agree this should get some more love. Feel free to do any contributions to `pixi` related to your command issue. The way described in #248 is still something we would like to support.
